app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -2,92 +2,95 @@
 <resources>
     <string name="app_name">Look4Sat</string>
     <string name="app_timer">00:00:00</string>
-    <string name="app_version">版本: %s</string>
-    <string name="app_subtitle">卫星追踪器</string>
+    <string name="app_version">版本：%s</string>
+    <string name="app_subtitle">人造卫星追踪和过境预测</string>
+
+    <!--Buttons-->
+
     <string name="btn_add">添加</string>
-    <string name="btn_accept">接受</string>
+    <string name="btn_accept">确定</string>
     <string name="btn_cancel">取消</string>
     <string name="btn_back">后退</string>
     <string name="btn_modes">模式</string>
     <string name="btn_select">全选</string>
     <string name="btn_done">完毕</string>
-    <string name="btn_clear">清除</string>
-    <string name="btn_clear_all">全部清除</string>
+    <string name="btn_clear">清空</string>
+    <string name="btn_clear_all">全部清空</string>
     <string name="btn_refresh">刷新</string>
     <string name="btn_map">地图</string>
     <string name="btn_filter">筛选</string>
     <string name="btn_satellites">卫星</string>
     <string name="btn_settings">设置</string>
-    <string name="btn_prev">上一页</string>
+    <string name="btn_prev">上一个</string>

```

Comment:
I found the botton is a little narrow for display 3 Chinese characters(equals to 6 letters), and now it can display 4 letters(2 Chinese characters).
@rt-bishop Cound we modify the width of buttons `btn_prev` and `btn_prev`?

Reply:
Hey @mrchi! Thanks for the PR, it's amazing to have people participating in the project. Feel free to adjust the size of the buttons and to make changes to the code in general. I'm going to review everything anyway after the PR gets merged.